astropy/timeseries/periodograms/lombscargle/implementations/fastchi2_impl.py

```diff
@@ -45,6 +46,13 @@ def lombscargle_fastchi2(
         of the input data. This is especially important if ``fit_mean = False``
     nterms : int, optional
         Number of Fourier terms in the fit
+    algorithm : str, optional
+        Referenced only if use_fft is true.
```

Comment:
Is this what you mean ?
```suggestion
        This option is ignored if if use_fft is False.
```